travis/script-syntax.py

```diff
@@ -19,14 +16,24 @@ def main():
         if '.git' in parts or 'depends' in parts:
             continue
         for filename in filenames:
-            if not filename.endswith(ext):
+            if not filename.endswith(args.ext):
                 continue
             full_path = os.path.join(cur, filename)
             try:
                 subprocess.check_output(cmd + [full_path])
             except subprocess.CalledProcessError:
                 err = True
+            except IOError:
+                # catch error if not in Travis and Lua/Ruby is not available
```

Comment:
All right, that makes sense. +1 to the change in 193e71793a3a0bf3e72b936a65059eec17c3f669